docs/configuration/configuration.md

```diff
@@ -754,6 +754,7 @@ Available meta labels:
 * `__meta_kubernetes_pod_annotation_<annotationname>`: Each annotation from the pod object.
 * `__meta_kubernetes_pod_annotationpresent_<annotationname>`: `true` for each annotation from the pod object.
 * `__meta_kubernetes_pod_container_name`: Name of the container the target address points to.
+* `__meta_kubernetes_pod_container_port_init`: `true` if the container is an [InitContainer](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)
```

Comment:
You still have to remove the `_port` part and move it one line above.

Reply:
Oh, I'm sorry, I didn't realize this was still there :see_no_evil:

Reply:
No problem, this is what code reviews are for :-)